src/pages/Player.vue

```diff
@@ -578,6 +694,9 @@ export default {
     setScreenProperties() {
       // sets various properties based on the device screen
       this.playerHeight = document.getElementById("videoPlayer").clientHeight;
+
+      if (screen.availHeight > 1.5 * screen.availWidth) this.isPortrait = true;
+      else this.isPortrait = false;
```

Comment:
this logic was used somewhere else as well right? can put it in Utilities and use that method everywhere.